### ISSUE TITLE
5.2.2: sphinx warnings

### ISSUE THREAD

**Issue by kloczek** [python/importlib_resources#235]
```console
+ /usr/bin/python3 setup.py build_sphinx -b man --build-dir build/sphinx
running build_sphinx
Running Sphinx v4.1.2
making output directory... done
building [mo]: targets for 0 po files that are out of date
building [man]: all manpages
updating environment: [new config] 4 added, 0 changed, 0 removed
reading sources... [100%] using
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
writing... importlib_resources.1 { using migration history } /home/tkloczko/rpmbuild/BUILD/importlib_resources-5.2.2/docs/using.rst:94: WARNING: py:mod reference target not found: importlib
/home/tkloczko/rpmbuild/BUILD/importlib_resources-5.2.2/docs/using.rst:129: WARNING: py:class reference target not found: pathlib.Path
/home/tkloczko/rpmbuild/BUILD/importlib_resources-5.2.2/docs/using.rst:141: WARNING: py:mod reference target not found: contextlib
/home/tkloczko/rpmbuild/BUILD/importlib_resources-5.2.2/docs/migration.rst:35: WARNING: py:class reference target not found: pathlib.Path
/home/tkloczko/rpmbuild/BUILD/importlib_resources-5.2.2/docs/migration.rst:55: WARNING: py:class reference target not found: contextlib.ExitStack
/home/tkloczko/rpmbuild/BUILD/importlib_resources-5.2.2/docs/migration.rst:65: WARNING: py:mod reference target not found: atexit
done
build succeeded, 6 warnings.
```

**Comment by jaraco** [comment]
Looks like these warnings also were implicated in jaraco/skeleton#36.

**Comment by jaraco** [comment]
Fixed in 8ea55f2fb26bd77997f0e9435bab2d41376a76d4.

**Comment by kloczek** [comment]
Tested.
Looks clean now :)
Thank you.
```console
+ /usr/bin/python3 setup.py build_sphinx -b man --build-dir build/sphinx
running build_sphinx
Running Sphinx v4.1.2
making output directory... done
loading intersphinx inventory from https://docs.python.org/3/objects.inv...
building [mo]: targets for 0 po files that are out of date
building [man]: all manpages
updating environment: [new config] 4 added, 0 changed, 0 removed
reading sources... [100%] using
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
writing... python-importlib_resources.3 { using migration history } done
build succeeded.
```
